src/sage/databases/cremona.py

```diff
@@ -1682,21 +1682,40 @@ def CremonaDatabase(name=None, mini=None, set_global=None):
         FeatureNotPresentError: database_should_not_exist_ellcurve is not available.
         '...db' not found in any of [...]
         ...Further installation instructions might be available at https://github.com/JohnCremona/ecdata.
+
+    Verify that :issue:`39072` has been resolved::
+
+        sage: C = CremonaDatabase(mini=False)  # optional - !database_cremona_ellcurve
+        Traceback (most recent call last):
+        ...
+        ValueError: full Cremona database is not available; run "sage -i database_cremona_ellcurve" to install it
```

Comment:
Technically `sage -i` doesn't work with meson https://doc-release--sagemath.netlify.app/html/en/installation/meson

Reply:
Isn't this the standard way to install optional packages?

Reply:
If not, what is the standard way?

Reply:
I don't know either (as in I know it works with the monolithic build system but I heard it supposedly doesn't work with the case it's built from meson)? Maybe ask @tobiasdiez

Reply:
Perhaps we should regard meson build as an alternative experimental build system...

Reply:
Yes, `sage -i` doesn't work with neither conda nor meson, and also not on some other systems. So it would indeed be nice to phrase this warning in a way to make sense across all of these supported ways to install sage. Perhaps just link to https://doc.sagemath.org/html/en/reference/spkg/database_cremona_ellcurve.html

Reply:
> Perhaps just link to https://doc.sagemath.org/html/en/reference/spkg/database_cremona_ellcurve.html

But that is even less descriptive. It explains neither `conda install` nor `sage -i`, and users may not know how to install optional package in the first place.

Can we just somehow make `sage -i` redirect to `conda install`?

Reply:
> It explains neither `conda install` nor `sage -i`

The first is because there is no package to install the cremona db via conda. The latter is indeed a "bug".

> Can we just somehow make `sage -i` redirect to `conda install`?

On arch, you would need to install https://aur.archlinux.org/packages/sage-data-cremona_ellcurve... (also missing btw from the docs page). I think such an automatic install would open a can of worms.

Reply:
I rephrased the message as @culler wished.